x/gov/bucket.go

```diff
@@ -101,6 +101,51 @@ func (b *ProposalBucket) Update(db weave.KVStore, id []byte, obj *Proposal) erro
 	return nil
 }
 
+const (
+	indexNameElectorate = "electorate"
+)
+
+// ResolutionBucket is the persistence bucket for resolutions.
+type ResolutionBucket struct {
+	orm.IDGenBucket
+}
+
+func NewResolutionBucket() *ResolutionBucket {
+	b := migration.NewBucket(packageName, "resolution", orm.NewSimpleObj(nil, &Resolution{})).
+		WithIndex(indexNameElectorate, indexElectorate, false)
+	return &ResolutionBucket{
+		IDGenBucket: orm.WithSeqIDGenerator(b, "id"),
+	}
+}
+
+func indexElectorate(obj orm.Object) ([]byte, error) {
```

Comment:
it is actually electorate index, similar to indexProposal. I named it like this for consistency. It is not a refID
index, because we only index the ID and not the version. So I think it is a valid name.

Reply:
I agree that this is returning a value used for an electorate index building. I think this function does not build the index and in my opinion, the current name does not represent what the function does.

It is a common practice to name indexes after the fields that are used to build it. In case of a single field index, I would expect a name like `field_name_idx`. That is why I think a name for those variables could be improved.

Reply:
I saw the name `chainAddressIndexer` in other parts of the code for such a function.
How about renaming to `electorateIDIndexer`?